app/engine/Flywheel.test.js

```diff
@@ -0,0 +1,342 @@
+  flywheel.pushValue(0.01116456)
+  flywheel.pushValue(0.011130263)
+  flywheel.pushValue(0.011082613)
+  flywheel.pushValue(0.011081761)
+  flywheel.pushValue(0.011062297)
+  flywheel.pushValue(0.011051853)
+  flywheel.pushValue(0.010973313)
+  flywheel.pushValue(0.010919756)
+  flywheel.pushValue(0.01086431)
+  flywheel.pushValue(0.010800864)
+  flywheel.pushValue(0.010956987)
+  flywheel.pushValue(0.010653396)
+  flywheel.pushValue(0.010648619)
+  flywheel.pushValue(0.010536818)
+  flywheel.pushValue(0.010526151)
+  flywheel.pushValue(0.010511225)
+  flywheel.pushValue(0.010386684)
+  testDeltaTime(flywheel, 0.011051853)
+  testSpinningTime(flywheel, 0.088970487)
+  testAngularPosition(flywheel, 9.42477796076938)
+  testAngularVelocity(flywheel, 95.26126661773489)
+  testAngularAcceleration(flywheel, 23.690349229418256)
+  testTorque(flywheel, 3.276505814703343)
+  testDragFactor(flywheel, 0.0001)
+  testIsDwelling(flywheel, false)
+  testIsUnpowered(flywheel, false)
+  testIsPowered(flywheel, true)
+  flywheel.pushValue(0.010769)
+  flywheel.pushValue(0.010707554)
+  flywheel.pushValue(0.010722165)
+  flywheel.pushValue(0.01089567)
+  flywheel.pushValue(0.010917504)
+  flywheel.pushValue(0.010997969)
+  flywheel.pushValue(0.011004655)
+  flywheel.pushValue(0.011013618)
+  flywheel.pushValue(0.011058193)
+  flywheel.pushValue(0.010807149)
+  flywheel.pushValue(0.0110626)
+  flywheel.pushValue(0.011090787)
+  flywheel.pushValue(0.011099509)
+  flywheel.pushValue(0.011131862)
+  flywheel.pushValue(0.011209919)
+  testDeltaTime(flywheel, 0.01089567)
+  testSpinningTime(flywheel, 0.24984299900000007)
+  testAngularPosition(flywheel, 25.132741228718345)
+  testAngularVelocity(flywheel, 96.59818534009558)
+  testAngularAcceleration(flywheel, -28.68758647905641)
+  testTorque(flywheel, -1.9356377068056956)
+  testDragFactor(flywheel, 0.0001)
+  testIsDwelling(flywheel, false)
+  testIsUnpowered(flywheel, true)
+  testIsPowered(flywheel, false)
+  flywheel.pushValue(0.020769)
+  flywheel.pushValue(0.020707554)
+  flywheel.pushValue(0.020722165)
+  flywheel.pushValue(0.02089567)
+  flywheel.pushValue(0.020917504)
+  flywheel.pushValue(0.020997969)
+  flywheel.pushValue(0.021004655)
+  flywheel.pushValue(0.021013618)
+  flywheel.pushValue(0.021058193)
+  flywheel.pushValue(0.020807149)
+  flywheel.pushValue(0.0210626)
+  flywheel.pushValue(0.021090787)
+  flywheel.pushValue(0.021099509)
+  flywheel.pushValue(0.021131862)
+  flywheel.pushValue(0.021209919)
+  testDeltaTime(flywheel, 0.02089567)
+  testSpinningTime(flywheel, 0.45261901530000004)
+  testAngularPosition(flywheel, 40.84070449666731)
+  testAngularVelocity(flywheel, 82.8344307883191)
+  testAngularAcceleration(flywheel, -31.179572048417487)
+  testTorque(flywheel, -2.431802912439266)
+  testDragFactor(flywheel, 0.0001)
+  testIsDwelling(flywheel, true)
+  testIsUnpowered(flywheel, true)
+  testIsPowered(flywheel, false)
+})
+
+// Test behaviour for noisy upgoing flank
```

Comment:
Are these placeholders for additional tests that we should add in the future? In that case add a `todo: add test case for ...` so we don't miss them when elaborating. If it means something else it would be good if you could make the comment more meaningful.